docs/_articles/en/release-information.md

```diff
@@ -5,6 +5,42 @@ lang: en
 
 ### To update the plugin, please follow the linked  [instructions](./en/getting-started/install/#upgrade-plug-in).
 
+## [v2.9.1](https://github.com/forcedotcom/sfdx-scanner/tree/v2.9.1) (05-21-2021)
+
+[Full Changelog](https://github.com/forcedotcom/sfdx-scanner/compare/v2.9.0...v2.9.1)
+
+### Release Summary
+
+* RetireJS will now work offlinbe
+* Underlying PMD Engine is upgraded to 6.34.0 (from 6.33.0) 
+* Bug fixes
+
+**Merged pull requests:**
+
+- @W-9296240@ Switch library used to detect binary files [\#411](https://github.com/forcedotcom/sfdx-scanner/pull/411)
+- @W-9282230@ Fixing changelog to update 2.8 release information [\#409](https://github.com/forcedotcom/sfdx-scanner/pull/409)
```

Comment:
Not sure if we need this. Pls let me know if you think I should remove this.

Reply:
Agree, this change is too meta and I feel it can be removed too.